docs/src/getting-started/setup.md

```diff
@@ -0,0 +1,64 @@
+# Setup instructions for the Fuel SDK
+
+## What you will need on your machine
+
+- Rust 2021 ([see here](https://doc.rust-lang.org/cargo/getting-started/installation.html))
+- A clone of the `fuel-indexer` repository:
+
+```sh
+
+git clone https://github.com/FuelLabs/fuel-indexer
+
+```
+
+- Install the following `cargo` crates:
+
+```sh
+
+cargo install forc fuel-core
+
+```
+
+- `forc` is the crate that holds the Sway language and Fuel's equivalent of `cargo`
+- `fuel-core` is the crate that contains the Fuel node software and execution
+
+## System Requirements
+
+There are several system requirements including llvm, clang and postgres.
+
+### Ubuntu
+
+```bash
+
+apt update
+apt install -y cmake pkg-config libssl-dev git gcc build-essential git clang libclang-dev llvm libpq-dev
+
+```
+
+### MacOS
+
+```bash
+
+brew update
+brew install openssl cmake llvm libpq postgresql
+
+```
+
+### Debian
+
+```bash
+
+apt update
+apt install -y cmake pkg-config libssl-dev git gcc build-essential git clang libclang-dev llvm libpq-dev
+
+```
+
+### Arch
+
+```bash
+
+pacman -Syu --needed --noconfirm cmake gcc openssl-1.0 pkgconf git clang llvm11 llvm11-libs postgresql-libs
+export OPENSSL_LIB_DIR="/usr/lib/openssl-1.0";
+export OPENSSL_INCLUDE_DIR="/usr/include/openssl-1.0"
```

Comment:
These aren't permanent. What's the purpose of setting these variables?
```suggestion
export OPENSSL_LIB_DIR="/usr/lib/openssl-1.0"
export OPENSSL_INCLUDE_DIR="/usr/include/openssl-1.0"
```

Reply:
@adlerjohn These are just copied over from the ReadMe. I'm assuming @tjsharp1 had a good reason to document them

Reply:
Hmmm. Well we can leave them in for now if there's no resolution and remove them later. If this PR is merged as-is then please file an issue to track investigating these lines.